dev-lua/lua-argparse/lua-argparse-0.6.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Command line argument parser for the Lua Programming Language"
+HOMEPAGE="https://github.com/mpeterv/argparse"
+SRC_URI="https://github.com/mpeterv/argparse/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc luajit"
+
+RDEPEND="
+	>=dev-lang/lua-5.1:=
+	luajit? ( dev-lang/luajit:2 )
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${PN//lua-/}-${PV}"
+
+DOCS=( README.md CHANGELOG.md )
```

Comment:
default docset `einstalldocs` installs already contains some files.

code from `phase-helpers.sh`
```bash
if ___eapi_has_einstalldocs; then
        einstalldocs() {
                (
                        if ! declare -p DOCS &>/dev/null ; then
                                local d
                                for d in README* ChangeLog AUTHORS NEWS TODO CHANGES \
                                                THANKS BUGS FAQ CREDITS CHANGELOG ; do
                                        [[ -f ${d} && -s ${d} ]] && docinto / && dodoc "${d}"
                                done
                        elif ___is_indexed_array_var DOCS ; then
                                [[ ${#DOCS[@]} -gt 0 ]] && docinto / && dodoc -r "${DOCS[@]}"
                        else
                                [[ ${DOCS} ]] && docinto / && dodoc -r ${DOCS}
                        fi
                )

                (
                        if ___is_indexed_array_var HTML_DOCS ; then
                                [[ ${#HTML_DOCS[@]} -gt 0 ]] && \
                                        docinto html && dodoc -r "${HTML_DOCS[@]}"
                        else
                                [[ ${HTML_DOCS} ]] && \
                                        docinto html && dodoc -r ${HTML_DOCS}
                        fi
                )
        }
fi
```

It's better to define `local DOCS=( blah bleh)` in `src_install` just before you run einstalldocs.

Reply:
fixed